openff/toolkit/topology/topology.py

```diff
@@ -2140,6 +2139,14 @@ def from_openmm(cls, openmm_topology, unique_molecules=None):
 
         from openff.toolkit.topology.molecule import Molecule
 
+        # TODO: Simply decorate with @requires_package("openmm") when simtk compatibility no longer needed
+        try:
+            from openff.units.openmm import from_openmm
+            from openmm import app
+        except ImportError:
+            from openff.units.simtk import from_simtk as from_openmm
+            from simtk.openmm import app
```

Comment:
We should be good to pull the trigger on this. Though - an interesting thought - this method doesn't actually require openmm until the very end for a unit conversion with the box vectors. So, if the `OFFTop.box_vectors` setter has openff-units magic to handle an openmm.unit.Quantity, then this entire method could run without a hard dep on openmm!

Reply:
I split this off into a self-contained PR: https://github.com/openforcefield/openff-toolkit/pull/1144